### PR TITLE
fix: url homepage.

### PR DESCRIPTION
**Pull request by ceoaliongroo**
#306

homepage

### CODE REVIEW

**Comment by ceoaliongroo** [comment]
Created test feature locally
screen shot 2015-01-28 at 1 42 23 pm

File: behat/features/bootstrap/FeatureContext.php [hunk 1]
```diff
@@ -81,6 +81,15 @@ public function iShouldSeeThanAMarkerDisappeared() {
   }
 
   /**
+   * @Then I should see login page
+   */
+  public function iShouldSeeLoginPage()
+  {
```
Comment (amitaibu) [inline behat/features/bootstrap/FeatureContext.php]: move up

File: behat/features/router.feature [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  Scenario: Authenticate user have to see homepage when set url "/"
+    Given I login with user "carlos"
+    When I visit "/"
+    Then I should see all the markers
+
+  @javascript @wip
+  Scenario: Anonymous user have to see login page when set url "/"
+    Given I am an anonymous user
+    When I visit "/"
+    Then I should see login page
```
Comment (amitaibu) [inline behat/features/router.feature]: ``see the login page``

**Comment by amitaibu** [comment]
Merged.

**Pull request closed without merging**